src/components/pages/GameContainer/Game/Pregame/Pregame.tsx

```diff
@@ -341,7 +341,7 @@ const Pregame = (props: PregameProps): React.ReactElement => {
           <ProTip
             message={'This is your chance to let your creativity shine!'}
           />
-          <h1>The Lobby is filling up...</h1>
+          <h2>The Lobby is filling up...</h2>
```

Comment:
I changed this to h1 because it's the first header on the page when rendered. Please keep it h1

Reply:
Changed it and pushed.